crates/re_viewer_context/src/space_view/space_view_class_registry.rs

```diff
@@ -37,15 +34,19 @@ impl SpaceViewClassRegistry {
         Ok(())
     }
 
-    /// Queries a space view type by name.
-    pub fn get(
-        &self,
-        name: SpaceViewClassName,
-    ) -> Result<&dyn DynSpaceViewClass, SpaceViewClassRegistryError> {
-        self.0
-            .get(&name)
-            .map(|boxed| boxed.as_ref())
-            .ok_or(SpaceViewClassRegistryError::TypeNotFound(name))
+    /// Queries a Space View type by class name.
+    fn get(&self, name: SpaceViewClassName) -> Option<&dyn DynSpaceViewClass> {
+        self.0.get(&name).map(|boxed| boxed.as_ref())
+    }
+
+    /// Queries a Space View type by class name and logs if it fails.
+    pub fn get_and_log_error(&self, name: SpaceViewClassName) -> Option<&dyn DynSpaceViewClass> {
+        let result = self.get(name);
+        // TODO(wumpf): Workaround for tensor not yet ported
+        if result.is_none() && name != "Tensor" {
+            re_log::error_once!("Unknown space view class \"{}\"", name);
```

Comment:
```suggestion
            re_log::error_once!("Unknown space view class {:/}", name);
```

https://github.com/rerun-io/rerun/blob/main/CODE_STYLE.md#misc for motivation

Reply:
https://github.com/rerun-io/rerun/pull/2332